hari_client/client/client.py

```diff
@@ -2116,3 +2371,194 @@ def get_visualisation_configs(
+            user_group: The user group for scoping the development set (default: None).
+
+        Returns:
+            models.DevelopmentSetResponse: The newly created development set object.
+        """
+        body = {
+            "name": name,
+            "training_attributes": training_attributes,
+            "user_group": user_group,
+        }
+
+        return self._request(
+            "POST",
+            "/trainingSets",
+            json=body,
+            success_response_item_model=models.DevelopmentSetResponse,
+        )
+
+    def get_aint_models(
+        self,
+    ) -> list[models.MlAnnotationModel]:
+        """
+        Retrieve all AI Nano Task (AINT) models.
+
+        Returns:
+            list[models.MlAnnotationModel]: A list of AI annotation model objects.
+        """
+        return self._request(
+            "GET",
+            f"/mlAnnotationModels",
+            # no params
+            success_response_item_model=list[models.MlAnnotationModel],
+        )
+
+    def get_aint_model(self, aint_model_id: str) -> models.MlAnnotationModel:
+        """
+        Retrieve a specific AI Nano Task (AINT) model by its ID.
+
+        Args:
+            aint_model_id: The unique identifier of the AI annotation model.
+
+        Returns:
+            models.MlAnnotationModel: The requested AI annotation model object.
+        """
+        return self._request(
+            "GET",
+            f"/mlAnnotationModels/{aint_model_id}",
+            # no params
+            success_response_item_model=models.MlAnnotationModel,
+        )
+
+    def train_aint_model(
+        self, name: str, development_set_id: str, user_group: str | None = None
+    ) -> models.MlAnnotationModel:
+        """
+        Train a new AI Nano Task (AINT) model on a specified development set.
+
+        Args:
+            name: A descriptive name for the AI model.
+            development_set_id: The unique identifier of the development set to use for training.
+            user_group: The user group for scoping this model (default: None).
+
+        Returns:
+            models.MlAnnotationModel: The newly trained AI annotation model.
+        """
+
+        body = {"name": name, "training_set_id": development_set_id}
+
+        return self._request(
+            "POST",
+            "/mlAnnotationModels",
+            json=body,
+            success_response_item_model=models.MlAnnotationModel,
+        )
+
+    def get_ai_annotation_runs(
+        self,
+    ) -> list[models.AiAnnotationRun]:
+        """
+        Retrieve all AI annotation runs.
+
+        Returns:
+            list[models.AiAnnotationRun]: A list of AI annotation run objects.
+        """
+        return self._request(
+            "GET",
+            f"/aiAnnotationRuns",
+            # no params
+            success_response_item_model=list[models.AiAnnotationRun],
+        )
+
+    def get_ai_annotation_run(self, run_id: str) -> models.AiAnnotationRun:
```

Comment:
```suggestion
    def get_ai_annotation_run(self, ai_annotation_run_id: str) -> models.AiAnnotationRun:
```
and everywhere else it's used, just for better readability